site/source/_posts/2017/11/17-Dojo-2-Beta-4/Dojo-2-Beta-4.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Dojo 2 Beta 4
+date: 2017-11-17 18:00:00
+authorId: kitsonk
+featured_image: featured.png
+---
+
+The Beta 4 Release of Dojo 2 puts us on a clear path to releasing Dojo 2.  It has several significant changes and updates.
```

Comment:
It has several significant changes and updates -> It has several significant changes and updates from earlier Dojo 2 beta releases. Here we highlight the current state of Dojo 2, as well as emphasize the recent changes and additions in beta 4.